runtime/compiler/x/runtime/X86Unresolveds.nasm

```diff
@@ -989,6 +990,20 @@ interpreterStaticAndSpecialGlue:
       test        cl, J9TR_MethodNotCompiledBit
       jnz         j2iTransition
 
+      ; Call to adjust trampoline reservations, since this callsite is now resolved
+      mov         edx, dword [rsi+20]
+      push        rdx                                          ; descriptor+24: CP index
+      mov         rdx, qword [rdi + J9TR_MethodFlagsOffset]
```

Comment:
Shouldn't his be `mov rdx, dword [rsi+12]` to get to the cpAddr?

Reply:
It is, but this also works, because `%rdi` is the `J9Method` addres.s and `J9TR_MethodFlagsOFfset` is the offset to constant pool from start of `J9Method`

Reply:
`mov         edx, dword [rsi+20]` looks ok to me and I have no clue why the address is invalid.
Is it always invalid, or only in some particular cases? maybe you could trace it instruction by instruction using the debugger and see how registers change